docs/tutorials/aws.md

```diff
@@ -891,9 +891,14 @@ For any given DNS name, only **one** of the following routing policies can be us
 - Latency-based routing: `external-dns.alpha.kubernetes.io/aws-region`
 - Failover:`external-dns.alpha.kubernetes.io/aws-failover`
 - Geolocation-based routing:
-  - `external-dns.alpha.kubernetes.io/aws-geolocation-continent-code`
```

Comment:
with 2 spaces, the rendering of the nested list doesn't happen correctly.
![CleanShot 2025-06-09 at 01 43 05](https://github.com/user-attachments/assets/b0284a95-2d41-4d52-8b0b-2ef3b461a175)

Reply:
so I updated the indent from 2 spaces to 4 spaces. But that made the linter sad. :(
I will change it back to 2 spaces just to make linter pass.